src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,250 @@
+- **Know** what element to use to use depending on the purpose of the text.
+- **Ensure** we are communicating the intended meaning of the enclosed text to the users.
+
+<span id="creating-elements-using-tags" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h4"
+  additionalClasses="sprk-u-mbs sprk-u-Measure"
+  variant="displayFour"
+>
+  Creating Elements Using Tags
+</SprkHeading>
+
+Use **tags** to create an element. Tags delimit the start and end of elements in the markup.
+
+An element consists of three parts:
+
+- The **opening tag** that represents the start of the paragraph element.
+- The **content** enclosed inside the tags.
+- The **closing tag** that represents where the element ends.
+
+<span id="examples" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h5"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayFive"
+>
+  Example
+</SprkHeading>
+
+`<p>` Download the Rocket Homes mobile application on your mobile device to start looking for your dream home.`</p>`
+
+- The `<p>` represents the start of the paragraph element.
+- The content is the enclosed text (<i>"Download the Rocket.... " </i>) inside the tags.
+- The `</p>` represents where the element ends.
+
+#### Common HTML Elements For UX Writers
+
+Use the strong and emphasis elements, to ensure users can perceive the semantic meaning indicated by the visual treatment, even if they’re using assistive technology like a screen reader.
+
+- The **strong element** `<strong></strong>`, defines text with strong importance.
+- The **emphasis element** `<em></em>`, defines text with stressed emphasis.
+
+> Use the **bold element** `<b></b>` and the **italic or idiomatic text element** `<i></i>` for formatting and to display special types of text.
+
+- The **bold element** `<b></b>` , draws attention to the element's contents, without extra importance.
+
+- The **italic or idiomatic text element** `<i></i>`, defines a part of text in an alternate voice or mood.
+
+<span id="semantic-elements" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h2"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayTwo"
+>
+  Semantic Elements
+</SprkHeading>
+
+Using HTML Semantic elements ensure users can perceive the semantic meaning indicated by the visual treatment, even if they’re using assistive technology like a screen reader.
+
+- [The Strong Element](#the-strong-element)
+- [The Emphasis Element](#the-emphasis-element)
+
+> To view a list of all the HTML elements available to add semantic meaning to copy, refer to [MDN’s Semantic element list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element#inline_text_semantics).
+
+### The Strong Element
+
+The <strong>strong element</strong> indicates strong importance, seriousness, or urgency of the enclosed text inside the `<strong></strong>` tags.
+
+- By default web browsers, render the contents inside strong tags in **bold type**.
+
+#### Usage
+
+Use for:
```

Comment:
We don't need "use for" because the header above it already clearly implies that it's for usage. Let's remove this for every section.